docs/redis.md

```diff
@@ -0,0 +1,34 @@
+# Usage of Redis for cozy-stack
+
+
+It will be important to figure out if we use a cluster of Redis or only one.
+Assuming a cluster allows us to scale better in the future but for now we will focus on a single redis target for the stack.
```

Comment:
We don't to shard the data in redis for now (and probably for a long time). What we need is high availability, and it is given by master/slave. From a developer point of view, it's the same as a single node. The work will be for our devops.

Also, I'd like to have configuration to use separate redis instances for different use cases:

- one redis instance (well, one master and one slave) for caching
- one for locks
- one for the jobs system
- etc.